From d9722312f48fa463d00dd2b9b0be010f551895f5 Mon Sep 17 00:00:00 2001
From: Ganesh Maharaj Mahalingam <ganesh.mahalingam@intel.com>
Date: Fri, 25 Mar 2022 09:40:51 -0700
Subject: [PATCH 2/2] Switch default graph driver to overlay

podman will be primarily used rootless and given they recommend using
overlay as the default graphdriver for rootless, we might as well set
that as the default.

This also fixes the error.
ERRO[0000] User-selected graph driver "overlay" overwritten by graph driver "vfs" from database - delete libpod local files to resolve

Signed-off-by: Ganesh Maharaj Mahalingam <ganesh.mahalingam@intel.com>
---
 vendor/github.com/containers/storage/types/options.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vendor/github.com/containers/storage/types/options.go b/vendor/github.com/containers/storage/types/options.go
index 5421c02..941d98b 100644
--- a/vendor/github.com/containers/storage/types/options.go
+++ b/vendor/github.com/containers/storage/types/options.go
@@ -244,7 +244,7 @@ func getRootlessStorageOpts(rootlessUID int, systemOpts StoreOptions) (StoreOpti
 		}
 	}
 	if opts.GraphDriverName == "" {
-		opts.GraphDriverName = "vfs"
+		opts.GraphDriverName = "overlay"
 	}
 
 	if os.Getenv("STORAGE_OPTS") != "" {
-- 
2.37.2

